test/lib/requirements.js

```diff
@@ -26,4 +36,19 @@ contract('Requirements Interface', (accounts) => {
     });
 
   });
+
+  xdescribe('Modifying requirements', () => {
+
+    it('add attribute', async () => {
+
+    });
+
+    it('remove attribute', async () => {
+
+    });
+
+    it('can\'t modify when not authorised', async () => {
+
+    });
+  });
```

Comment:
For the future, I wouldn't include files that are half complete; This tests and the changes to the requirements don't add anything in the context of measurements;